class UserNotFound(Exception):
    pass

class UserPhoneNumberAlreadyExists(Exception):
    pass